release.py

```diff
@@ -129,21 +127,23 @@ def parse_args(argv):
                         help='no-sanity-checks; i.e. skip sanity checks commands')
     parser.add_argument('--no-generate-source-file', dest='no_source_file', action='store_true', default=False,
                         help='Do not generate source file when building')
-    parser.add_argument('--ignition-repo', dest='ignition_repo', action='store_true', default=False,
-                        help='use ignition robotics URL repositories instead of OSRF')
+    parser.add_argument('--no-ignition-auto', dest='no_ignition_auto', action='store_true', default=False,
+                        help='Use package name to create --package-alias if package name starts with ign-')
     parser.add_argument('--upload-to-repo', dest='upload_to_repository', default="stable",
                         help='OSRF repo to upload: stable | prerelease | nightly')
     parser.add_argument('--extra-osrf-repo', dest='extra_repo', default="",
                         help='extra OSRF repository to use in the build')
     parser.add_argument('--nightly-src-branch', dest='nightly_branch', default="main",
                         help='branch in the source code repository to build the nightly from')
     args = parser.parse_args()
+
+    IGN_AUTO_DETECTED = args.package.startswith('ign-') if not args.no_ignition_auto else False
+    print(IGN_AUTO_DETECTED)
     if not args.package_alias:
-        args.package_alias = args.package
+        args.package_alias = args.package.replace('ign-', 'ignition-') if IGN_AUTO_DETECTED else args.package
```

Comment:
nit: it's just my opinion but putting the logic all on one line makes it a bit harder to follow (this line and the one above)

Reply:
let's try in a different way: d4d3375